cdk/lib/cdk-stack.ts

```diff
@@ -52,16 +52,12 @@ export class CdkStack extends GuStack {
       description: "Lambda to notify about instances with missing tags",
       timeout: Duration.seconds(30),
       memorySize: 512,
```

Comment:
Can these two props be omitted now as they match the default values as of https://github.com/guardian/cdk/pull/188?

Reply:
Yep, good point, thanks for spotting this 👍